src/System.Net.Requests/src/System/Net/FtpDataStream.cs

```diff
@@ -56,6 +56,8 @@ protected override void Dispose(bool disposing)
             }
         }
 
+        //TODO: Add this to FxCopBaseline.cs once https://github.com/dotnet/roslyn/issues/15728 is fixed
```

Comment:
This is an internal type, so we could just change it from an explicit to an implicit interface implementation without impacting public surface area.